sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -627,6 +633,10 @@ export interface BlobSyncCopyFromURLOptions extends CommonOptions {
    * Optional. Version 2019-07-07 and later.  Specifies the name of the encryption scope to use to encrypt the data provided in the request. If not specified, encryption is performed with the default account encryption scope.  For more information, see Encryption at Rest for Azure Storage Services.
    */
   encryptionScope?: string;
+  /**
+   * Optional. Default 'replace'.  Indicates if source tags should be copied or replaced with the tags specified by {@link tags}.
```

Comment:
"@link" is not correct.

Reply:
It can directly to the "tags" option in this structure. Didn't see issue about it.